proto/substrait/algebra.proto

```diff
@@ -144,6 +144,14 @@ message ReadRel {
   }
 }
 
+// (Re)qualify column names in a relation. If the system does not need qualified names, this can be ignored.
+message SubqueryAliasRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  repeated string names = 3;
```

Comment:
If we have structured content would these names be in depth first form as described elsewhere?  If so, that comment should be copied here as well.

Reply:
This would be aliaising the subquery, not the field names - so this is more similar to the `names` on a NamedTable rather than the depth-first order used for fields.

Reply:
Based on that it sounds like this is a list of names that all refer to the input relation.